docs/manual/src/python/setup.md

```diff
@@ -0,0 +1,236 @@
+# Intro
+
+The main idea is to build the bindings with:
+
+python .\setup.py bdist_wheel --verbose ;
+$wheelFile = Get-ChildItem -Path .\dist\ -Recurse -Include * ;
+pip3 install $wheelFile --force-reinstall ;
+
+Then you must create a setup.py file. This file will include the command to generate python .py bindings, link .dll and then build to pipy. Requires python version greater than 3.6.
+
+
+The full example is available at this [address](https://github.com/mozilla/uniffi-rs/tree/main/examples/distributing/uniffi-rust-to-python-library/src/setup.py). In order to reproduce the issue, let's see some points about the setup.py file.
+
+
+# Create setup.py file.
+## rustc
+
+get rustc version from your setup.py file:
```

Comment:
Please remove most of the code from this file - there's no need to duplicate it between the markdown and the new example.